test/Interop/Cxx/operators/member-inline-silgen.swift

```diff
@@ -12,3 +12,13 @@ public func sub(_ lhs: inout LoadableIntWrapper, _ rhs: LoadableIntWrapper) -> L
 // CHECK: end_access [[SELFACCESS]] : $*LoadableIntWrapper
 
 // CHECK: sil [clang LoadableIntWrapper."-"] [[NAME]] : $@convention(c) (@inout LoadableIntWrapper, LoadableIntWrapper) -> LoadableIntWrapper
+
```

Comment:
And a SILGen test for the address only type, please!

Reply:
Done!